README.rst

```diff
@@ -131,6 +131,17 @@ Stingray uses `py.test` for testing. To run the tests, try::
 
    $ python setup.py test
 
+If you have installed Stingray via pip or conda, the source directly might 
```

Comment:
"the source ~directly~ might not be easily accessible"

Reply:
eek, source *directory*, I swear this cold has made me into a zombie.

Reply:
:)